### DISCUSSION TITLE
Модерирование

### DISCUSSION THREAD

**Discussion by Falensted** [LacunaHub/.github#21] [Предложения]
Предлагаю в авто модерации добавить функцию которая удалит все слова кроме перечисленных что то похоже на функцию с фильтраций ссылок или слов (в фильтраций слов есть только реестр запрещённых слов)  для чего такая функция ? ну например в моём случае в канале где предназначен для комманд бота не хочу что бы писали лишнее а писали только комманды предназначенные для бота возможна такая функция пригодиться и другим у меня пока такая идея.

**Comment by rovius** [comment]
Слишком специфический use case, причём как в части фильтрации, так и в части канала исключительно для команд.
